cypress/integration/pages/searchSpec.ts

```diff
@@ -63,42 +63,15 @@ describe("Search", () => {
       }
     );
 
-    context("when a search has more results than the requested amount", () => {
```

Comment:
Why is this test no longer required?

Reply:
Good check. This test is required.
 With the changes in Search page, the test cases using mock service data were failing  which took me a while to figure out and in the process of trying out different options, I think I somehow deleted it.  With Mock server 'once' API call, the mocked data was not returning which I didn't understand why. ('once' API  is used to mark the current request handler as used and to ignore any subsequent request matches to this handler. )